docs/section5/Rule5-51.md

```diff
@@ -0,0 +1,30 @@
+
+# Envelope - Rule 5-51  
+
+**Rule ID:** 5-51  
+**Rule Description:** Shading by adjacent structures and terrain is the same in the proposed design as in user model.  
+**Rule Assertion:** P-RMR building:has_site_shading = U-RMR building:has_site_shading  
+**Appendix G Section:** Section G3.1-14(a) Building Envelope Modeling Requirements for the Proposed design  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:** None  
+
+## Rule Logic:  
+
+**Rule Assertion:**  
+
+- Case 1: If shading by adjacent structure and terrain is modeled the same in P-RMR as in U-RMR: `if P_RMR.building.has_site_shading == U_RMR.building.has_site_shading: PASS`
+
+- Case 2: Else: `else: FAIL`
+
+**[Back](../_toc.md)**
+
+**Notes:**
+
+1. Did not find building.has_site_shading in the current schema.
```

Comment:
Same comments may apply as to rule 50, but perhaps what's included is enough for this rule.

Reply:
Updated using building.has_site_shading.